README.rst

```diff
@@ -73,7 +149,7 @@ If you'd like more details around installing PyMedPhys go to the
 ..
     TODO: reword:
 
-The PyMedPhys documentation is split into four categories:
+The PyMedPhys documentation is split into five categories:
```

Comment:
Would you mind merging the [fix-readme-links](https://github.com/pymedphys/pymedphys/tree/fix-readme-links) branch (see #1699) into this one and adjusting for the changes? That PR contains some changes that will otherwise conflict with yours here.

Reply:
@Matthew-Jennings Sorry, I did not see that (see https://github.com/pymedphys/pymedphys/pull/1699) before I edited my README.rst. Yes, that PR and my PR indeed have conflicts. It's best for me to work based on your PR.

Reply:
@Matthew-Jennings I just edited the PR (https://github.com/pymedphys/pymedphys/pull/1699) by adding "statement of need" and mentioned "how-to guides" page includes examples for users.

Reply:
No need to be sorry!

I don't mind which way you go about it: editing the existing PR or merging changes from that PRs branch into this one.

On reflection, it was probably premature of me to ask you to merge in the changes from the other one anyway. I expected it to make it into `main` quickly.